src/quo2/components/buttons/button/type_values.cljs

```diff
@@ -0,0 +1,186 @@
+(ns quo2.components.buttons.button.type-values
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo2.theme]))
```

Comment:
We can drop the version number from the alias and just use `quo.theme`, unless of course we have competing old + new namespaces on the same file.